### PR TITLE
Initial code commit

### PR DESCRIPTION
**Pull request by jonyo**

### CODE REVIEW

File: src/Amplitude.php [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+     * Example:
+     * <code>
+     * // In user initialization section of app...
+     * Zumba\Amplitude\Amplitude::getInstance()->init('APIKEY','johnny@example.com')
+     *     ->addUserProperties(['name' => 'Johnny 5'])
+     *     ->logQueuedEvents();
+     *
+     * // Elsewhere in your app, this could happen before OR after the above initialization...
+     * // See queueEvent() for more info
+     * Zumba\Amplitude\Amplitude::getInstance->queueEvent(
```
Comment (nkcmr) [inline src/Amplitude.php]: @jonyo should it be `Zumba\Amplitude\Amplitude::getInstance()->queueEvent(` (missing `()`)

**Comment by jonyo** [comment]
Oops, code sniff errors, fixing...

File: src/Amplitude.php (hunk 1 above)
Comment (jonyo) [inline src/Amplitude.php]: @nkcmr thanks, good catch, will fix.

**Comment by jonyo** [comment]
PSR-2 issues fixed

**Comment by nkcmr** [comment]
@jonyo having to call `::getInstance()` then `->init` seems a bit redundant. what if it were instead just:

```php
$amp = Zumba\Amplitude\Amplitude::init('default', 'API_KEY');
```

**Comment by nkcmr** [comment]
@jonyo accidently closed.

**Comment by jonyo** [comment]
My bad, forgot to scan tests folder too for psr-2...

**Comment by jonyo** [comment]
And....  done.  Also @nkcmr the thing you brought up is fixed.

**Pull request merged**